pkg/minikube/config/types.go

```diff
@@ -43,6 +43,7 @@ type ClusterConfig struct {
 	HyperkitVpnKitSock      string   // Only used by the Hyperkit driver
 	HyperkitVSockPorts      []string // Only used by the Hyperkit driver
 	DockerEnv               []string // Each entry is formatted as KEY=VALUE.
+	VolumeMounts            []string
```

Comment:
this seems to be used only by Docker/Podman driver, change name to ContainerVolumeMounts and add comment that is used only by Docker/podman